composer.json

```diff
@@ -25,12 +25,12 @@
     },
     "require-dev": {
         "symfony-cmf/testing": "^1.3",
-        "matthiasnoback/symfony-dependency-injection-test": "~0.7,>=0.7.4",
+        "matthiasnoback/symfony-dependency-injection-test": "^0.7.4",
         "matthiasnoback/symfony-config-test": "^1.2.3",
-        "sonata-project/block-bundle": ">=2.2.8",
-        "sonata-project/core-bundle": ">=2.2.3",
+        "sonata-project/block-bundle": "^2.2.8",
+        "sonata-project/core-bundle": "^2.2.3",
         "sonata-project/doctrine-phpcr-admin-bundle": "~1.1",
-        "symfony-cmf/tree-browser-bundle": ">=1.0.0-RC1",
+        "symfony-cmf/tree-browser-bundle": "^1.0.0|^2.0.0",
```

Comment:
this is require-dev only. if we are not compatible with tree 2.0, we
would have to put a conflict line in composer.json, otherwise
installations could start to go wrong once tree 2.0 is released.

hm, but travis installs tree 1.1 anyways - i guess its because of
sonata. so i don't know why we have this explicitly in composer.json
anyways. we do not suggest it so no explanations what this could be
useful for apart from sonata admin.

ok if i simply remove this line?